### PR TITLE
[MRG+2] add gitter link in README

### PR DESCRIPTION
**Pull request by patniharshit**
refs #8186

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -61,11 +61,11 @@ matplotlib's communication channels include active mailing lists:
 * `Announcement  <https://mail.python.org/mailman/listinfo/matplotlib-announce>`_ mailing list: matplotlib-announce@python.org
 * `Development <https://mail.python.org/mailman/listinfo/matplotlib-devel>`_ mailing list: matplotlib-devel@python.org
 
-
 The first is a good starting point for general questions and discussions.
 
-
-
+.. image:: https://badges.gitter.im/Join%20Chat.svg
```
Comment (phobson) [inline README.rst]: Could you add a sentence that explains that gitter is for coordinating development and asking questions directly related to contributing to matplotlib?

File: README.rst [hunk 2]
```diff
@@ -61,11 +61,13 @@ matplotlib's communication channels include active mailing lists:
...
 
-
 The first is a good starting point for general questions and discussions.
 
+.. image:: https://badges.gitter.im/Join%20Chat.svg
+   :alt: Join the chat at https://gitter.im/matplotlib/matplotlib
+      :target: https://gitter.im/matplotlib/matplotlib?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
 
-
+The gitter is for coordinating development and asking questions directly related to contributing to matplotlib.
```
Comment (QuLogic) [inline README.rst]: "The gitter chat room" or just "Gitter".

File: README.rst [hunk 3]
```diff
@@ -61,11 +61,13 @@ matplotlib's communication channels include active mailing lists:
 * `Announcement  <https://mail.python.org/mailman/listinfo/matplotlib-announce>`_ mailing list: matplotlib-announce@python.org
 * `Development <https://mail.python.org/mailman/listinfo/matplotlib-devel>`_ mailing list: matplotlib-devel@python.org
 
-
 The first is a good starting point for general questions and discussions.
 
+.. image:: https://badges.gitter.im/Join%20Chat.svg
```
Comment (QuLogic) [inline README.rst]: I don't think we should repeat the image here; just have some text (like the alt text) pointing to the chat room.

**Comment by phobson** [comment]
Thanks @patniharshit !

**Comment by dstansby** [comment]
Is there any point backporting this since it only appears on github?

**Comment by QuLogic** [comment]
I think this corrects something that was already backported, but i don't feel super-strongly about it.

**Pull request merged**